terraform/deploy/app/rbac-db-init.ddl.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE IF NOT EXISTS `User` (
+    id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+    username VARCHAR (256) NOT NULL,
```

Comment:
Would it be possible to add another column 'account' varchar(256) default null. Since the ADFS users 'usernames' will effecitively be a random number, maintainence would be easier if we could note some identifying string against their username. In this case, the sync job between cognito and the DB could put the external ID into account as well as the username.